### ISSUE TITLE
Cannot Exit Text Box With Keyboard

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1693]
It is not possible (or at least not obvious) how to shift focus from the `CodeMirror` component, used for editing templates, with just a keyboard. We should investigate whether it's possible to add the ability to exit the text box with just keyboard input.
